R/ROOT_julia/build_tarballs.jl

```diff
@@ -0,0 +1,68 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ROOT_julia"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+   GitSource("https://github.com/JuliaHEP/ROOT.jl.git", "26217370e35ab78c56d87f36cc14778eb798c5be")
+]
+
+# See https://github.com/JuliaLang/Pkg.jl/issues/2942
+# Once this Pkg issue is resolved, this must be removed
+uuid = Base.UUID("a83860b7-747b-57cf-bf1f-3e79990d037f")
+delete!(Pkg.Types.get_last_stdlibs(v"1.6.3"), uuid)
+
+# needed for julia_versions
+include("../../L/libjulia/common.jl")
+
+#build for releases (excluding DEV and RC) only:
+supported_julia_versions = filter(v -> match(r"^\d+\.\d+\.\d+$", string(v)) !=nothing, julia_full_versions)
```

Comment:
Any particular reason to exclude the DEV & RC versions? They usually give you a head start for supporting a new Julia version.

If you really want to do this, then instead of parsing version numbers, why not use that the versions are already stored in parsed form?
```suggestion
supported_julia_versions = filter(v -> isempty(v.prerelease), julia_full_versions)
```

Reply:
I may not remember well. 

Testing it again, without the filter I get the error `ERROR: LoadError: ArgumentError: Invalid character in tag value "1.12.0-beta3"!`. So might actually the reason that conducted me to filter out theses releases.

Reply:
Ah, you should use `julia_versions` instead of `julia_full_versions` (the latter is for internal use of the libjulia recipe).